command/wrist.py

```diff
@@ -0,0 +1,81 @@
+import utils
+import constants
+import wpilib
+import config
+from oi.keymap import Controllers
+from toolkit.command import SubsystemCommand, radians
+from subsystem import Wrist
+from commands2 import SequentialCommandGroup
+import math
+
+class ZeroWrist(SubsystemCommand[Wrist]):
+
+    def __init__(self, subsystem: Wrist):
+        super().__init__(subsystem)
+        self.subsystem = subsystem
+
+    def initialize(self):
+        self.subsystem.zero_wrist()
+
+    def execute(self):
+        pass
+
+    def isFinished(self):
+        return self.subsystem.is_at_angle(0)
+    
+    def end(self, interrupted: bool):
+        if not interrupted:
+            self.subsystem.wrist_zeroed = True
+            utils.LocalLogger.debug("Wrist zeroed")
+
+class SetWrist(SubsystemCommand[Wrist]):
+
+    def __init__(self, subsystem: Wrist, angle: radians):
+        super().__init__(subsystem)
+        self.subsystem = subsystem
+        self.angle = angle
+
+    def initialize(self):
+        self.subsystem.set_wrist_angle(self.angle)
+
+    def execute(self):
+        pass
+
+    def isFinished(self):
+        return self.subsystem.is_at_angle(self.angle)
+
+    def end(self, interrupted:bool):
+        self.subsystem.set_wrist_angle(self.subsystem.get_wrist_angle()) #I copied this from the old repo's command, but I'm confused by the logic. pls explain when u see this! thx
+        utils.LocalLogger.debug("Wrist position " + str(self.target) + " acheived")
+
+class FeedIn(SubsystemCommand[Wrist]):
+    def __init__(self, subsystem: Wrist):
+        super().__init__(subsystem)
+
+    def initialize(self):
+        self.subsystem.feed_in()
+
+    def execute(self):
+        pass
+
+    def isFinished(self):
+        return self.subsystem.note_staged
+
+    def end(self):
+        utils.LocalLogger.debug("Fed-in")
+
```

Comment:
You need to run one more command for shooting, which stops when the beam break does not detect anymore. This command needs to feed the note through the flywheel, so it needs to have a different method that you need to make in the subsystem file, as well as a new config variable for the speed.

Reply:
1. What should the method be?
2. Isnt the config variable just feeder_velocity (config.py line 69)?
3. Which sensor is the beam break? I will also look at the old repo, but if I haven't figured it out by the time you see this, pls tell me